utils/benchmark/setupForTests.sh

```diff
@@ -8,9 +8,16 @@ SKELETON_URL="https://github.com/seznam/IMA.js-skeleton.git"
 NPM_LOCAL_REGISTRY_URL_NO_PROTOCOL="localhost:4873"
 NPM_LOCAL_REGISTRY_URL="http://${NPM_LOCAL_REGISTRY_URL_NO_PROTOCOL}/"
 
+cd ..
+
 ROOT_DIR=`pwd`
-CREATE_IMA_APP_DIR="$ROOT_DIR/packages/create-ima-app"
-PACKAGE_VERSION="0.0.0-next"
+ROOT_DIR_IMA="$ROOT_DIR/ima"
```

Comment:
We are hardcoding the repository name here. If we ever migrate to a different repo name, this will get broken.

Maybe we can find another way to do this? (Like calling `pwd` in this dir before we do `cd ..`)

Reply:
Newly set `ROOT_DIR_IMA` before `cd ..` command.